Calculator/Calculator/Model/ExpressionParser.swift

```diff
@@ -0,0 +1,22 @@
+//  Calculator - ExpressionParser.swift
+//  created by vetto on 2023/01/27
+import Foundation
+
+enum ExpressionParser {
+    static func parse(from input: String) -> Formula {
+        let operandsArray = componentsByOperators(from: input).compactMap { Double($0) }
+        let operatorsArray = input.compactMap{ Operator(rawValue: $0) }
```

Comment:
```suggestion
        let operatorsArray = input.compactMap { Operator(rawValue: $0) }
```
고차함수와 중괄호(`{`) 사이에는 띄어쓰기를 넣어주세요~!